### DISCUSSION TITLE
GooFace: Default to privacy?

### DISCUSSION THREAD

**Discussion by tomoverlund** [uBlockOrigin/uBlock-issues#2035] [General] [answered]
This has been bothering me for some time. There was a big hubbub when it was discovered that the Brave browser had [whitelisted](https://nakedsecurity.sophos.com/2019/02/12/privacy-browser-braves-user-concern-over-facebook-whitelist/) Facebook in the code. But one comment surprised me:

> "He added that Brave’s Facebook blocking is “at least as good” as uBlock origin, which is a cross-platform ad blocker."

I was shocked to learn this. Now, of course this isn't quite true -- uBlock allows you to set your own rules, but it is true for the default user experience. And this just seems wrong to me. These tech giants are privacy-destroying, mega-industrial sized hoovers of the Internet. I have no Facebook account, and it seems terrible that I'm tracked by default when using a plugin design to protect my privacy. I think the default should block these tech giants, with the option to enable them for those users that want them.

**Comment by u-RraaLL** [comment]
Are you trying to imply uBO doesn't block trackers by default?

**Comment by tomoverlund** [comment]
Facebook and Google are tracking and advertising companies. When you connect to them on practically every website, they in all likelihood are tracking you, even if the functionality isn't labeled as a "tracker". I do not have a Facebook account. There is no legitimate reason, then, that I should be connected to "connect.facebook.net".

**Comment by gorhill** [comment, marked as answer]
> There is no legitimate reason, then, that I should be connected to "connect.facebook.net".

There are filters for `connect.facebook.net` in EasyPrivacy. If you find pointless connections to `connect.facebook.net`, just report them -- vague, unspecific claims are unactionable.

**Reply by tomoverlund** [reply]
I figured something so basic must be functioning as designed.

**Reply by tomoverlund** [reply]
I'm glad this was closed, because it pinged me via email, and I had forgotten where I had written this comment. For funzies, try and figure out why the following google.com sites are greenlit in default uBlock when visiting www.foxnews.com:

contributor.google.com
www.google.com

My lesson from the last exchange years ago was to use "hard mode", and manage my own whitelists.
